src/nvim/path.c

```diff
@@ -1098,7 +1102,7 @@ int gen_expand_wildcards(int num_pat, char_u **pat, int *num_file,
     p = pat[i];
 
     if (vim_backtick(p))
-      add_pat = expand_backtick(&ga, p, flags);
+      add_pat = (int)expand_backtick(&ga, p, flags);
```

Comment:
> expand_backtick originally returned an int and I changed the return type.

Indeed, hence my suggestion. The return value of `expand_backtick`, since it's a `size_t` isn't guaranteed to fit into an `int`.